ftp-native/src/main/java/org/ballerinalang/stdlib/ftp/util/FTPUtil.java

```diff
@@ -56,7 +56,7 @@ private FTPUtil() {
     }
 
     public static boolean notValidProtocol(String url) {
```

Comment:
If we use an enum for this we can remove this method, since the validation is done from Ballerina side.

Reply:
Removed.